Hrmanagement.DataModel/ViewModel/leaveTypeVm.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hrmanagement.DataModel.ViewModel
+{
+    public class leaveTypeVm
+    {
+        public int LeaveTypeId { get; set; }
+
+        public string? LeaveTypes { get; set; }
```

Comment:
Why LeaveTypes. Do you plan to pass multiple values (i.e. comma separated) in this ?